django/core/files/utils.py

```diff
@@ -18,6 +18,7 @@ def __init__(self, file):
     readline = property(lambda self: self.file.readline)
     readlines = property(lambda self: self.file.readlines)
     seek = property(lambda self: self.file.seek)
+    seekable = property(lambda self: self.file.seekable)
```

Comment:
A subtle but important distinction, thanks for pointing that out.  I've updated it to add the version check.